src/main/java/appeng/util/Platform.java

```diff
@@ -138,6 +139,11 @@
 
 	public static final int DEF_OFFSET = 16;
 
+	// public static final IStorageChannel<IAEItemStack> ITEM_CHANNEL = AEApi.instance().storage().getStorageChannel(
+	// IItemStorageChannel.class );
+	// public static final IStorageChannel<IAEFluidStack> FLUID_CHANNEL = AEApi.instance().storage().getStorageChannel(
+	// IFluidStorageChannel.class );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)